src/app/test-record/test-record.mapper.ts

```diff
@@ -0,0 +1,288 @@
+import { Injectable } from '@angular/core';
+import { TestResultModel } from '@app/models/test-result.model';
+import { TEST_STATION_TYPE } from '@app/test-record/test-record.enums';
+import { TestRecordTestType } from '@app/models/test-record-test-type';
+import { TestType } from '@app/models/test.type';
+import { COUNTRY_OF_REGISTRATION } from '@app/app.enums';
+
+export interface TestTypesApplicable {
+  seatBeltApplicable: { number: string };
+  defectsApplicable: { number: string };
+  emissionDetailsApplicable: { number: string };
+  testSectionApplicable1: { number: string };
+  testSectionApplicable2: { number: string };
+}
+
+@Injectable({ providedIn: 'root' })
+export class TestRecordMapper {
+  getTestStationType(type: string) {
+    switch (type) {
+      case 'atf':
+        return TEST_STATION_TYPE.ATF;
+        break;
+      case 'gvts':
+        return TEST_STATION_TYPE.GVTS;
+        break;
+      case 'hq':
+        return TEST_STATION_TYPE.HQ;
+        break;
+    }
+  }
+
+  getTestTypeApplicable(sectionName: string) {
+    const testTypesApplicable = {
```

Comment:
Can this be stored in a config file with enums instead?
I believe this could be reused for unit tests andpossible other vehicle types?